libcontainer/seccomp/config.go

```diff
@@ -111,3 +112,35 @@ func ConvertStringToArch(in string) (string, error) {
 	}
 	return "", fmt.Errorf("string %s is not a valid arch for seccomp", in)
 }
+
+// List of flags known to this version of runc.
+var flags = []string{
+	"SECCOMP_FILTER_FLAG_TSYNC",
```

Comment:
Does this need to be added to the spec?

Reply:
We recognize this flag, as it is a valid flag, but we ignore it, since in our context it does not make sense. This is why it is not added to the spec (since it does not do anything for the container).